src/fromJS.js

```diff
@@ -13,12 +14,25 @@ export function fromJS(value, converter) {
   );
 }
 
+function getToSeq(value) {
+  if (Array.isArray(value)) {
+    return IndexedSeq;
+  }
+
+  if (value instanceof Set) {
+    return ImmutableSet;
```

Comment:
This should be determining a kind of Seq to use

```suggestion
    return SetSeq;
```